ziggurat-core-crawler/src/summary.rs

```diff
@@ -20,9 +22,9 @@ pub struct NetworkSummary {
     /// Crawler's runtime.
     pub crawler_runtime: Duration,
     /// Addresses of good nodes.
-    pub node_ips: Vec<String>,
+    pub node_addrs: Vec<SocketAddr>,
```

Comment:
How about:
`nodes_addrs` 
just to emphasize that vector contains addresses for all nodes not one node's addresess. 
Up to your decision.

Reply:
hmm, sorry sir, I think I'm going to keep it as it is:  each item in the vector is a node address, or `node_addr`.  A group of node addresses, for me, would be `node_addrs`.  `node` is the adjective, modifier, should not be in plural, while `addr` is the thing.
And I think people using the library, know that each node (as we are using/defining them) have a unique address, which is even a key.